jobs/scripts/nightly-builds/nightly-builds.sh

```diff
@@ -9,11 +9,21 @@ artifact()
 # if anything fails, we'll abort
 set -e
 
+BUILDREQUIRES="libaio-devel librdmacm-devel libattr-devel libxml2-devel readline-devel openssl-devel libibverbs-devel fuse-devel glib2-devel userspace-rcu-devel libacl-devel sqlite-devel libuuid-devel"
+
+if [ ${CENTOS_VERSION} -eq 8 ]
+then
+    ENABLE_REPOS="--enablerepo=PowerTools,Devel"
+    BUILDREQUIRES="$BUILDREQUIRES python3-devel rpcgen libtirpc-devel"
+else
+    BUILDREQUIRES="$BUILDREQUIRES python-devel"
+fi
+
 # install basic dependencies for building the tarball and srpm
 yum -y install git autoconf automake gcc libtool bison flex make rpm-build mock createrepo_c
```

Comment:
I think it is required to install `epel-release` on CentOS-8 to get mock?

Reply:
Did you see this comment? I do not see where `epel-release` is installed. Maybe it is not needed, and `mock` is also available in an other repository? Please explain, thanks!

Reply:
@rkothiya I am still wondering about this part. Is `epel-release` not needed?

Reply:
I did not have to install that and without it on centos-8 and centos-7 machine I was able to run the script. But I can add it here.

Reply:
Ok, don't install it when there is no need to have it.

Reply:
epel-release might be needed here. Rinku, did you tested it out on fresh VM?

Reply:
@nixpanic Rinku tested them out of softserve machines which have already installed packaged as pre-req for regression test suite. I hope there is no other package required other than these? Can @rkothiya please check.

Reply:
I was expecting that `epel-release` needs to be installed to get a repository where `mock` is available. If it comes from either the `PowerTools` or `Devel` repositories, `epel-release` would not be required. Otherwise, it needs to be installed before installing the `${BUILDREQUIRES}`.

Reply:
Done

Reply:
@rkothiya Now, why have you removed epel-release. Can you please clarify here?

Reply:
I have not removed it but installed it before installing other packages.